packages/flutter/lib/src/cupertino/text_selection_toolbar_button.dart

```diff
@@ -170,4 +166,85 @@ class _CupertinoTextSelectionToolbarButtonState extends State<CupertinoTextSelec
       return child;
     }
   }
+
+  Widget _getContentWidget(BuildContext context) {
+    if (widget.child != null) {
+      return widget.child!;
+    }
+    final Widget textWidget = Text(
+      widget.text ?? CupertinoTextSelectionToolbarButton.getButtonLabel(context, widget.buttonItem!),
+      overflow: TextOverflow.ellipsis,
+      style: _kToolbarButtonFontStyle.copyWith(
+        color: widget.onPressed != null
+            ? _kToolbarTextColor.resolveFrom(context)
+            : CupertinoColors.inactiveGray,
+      ),
+    );
+    if (widget.buttonItem == null) {
+      return textWidget;
+    }
+    switch (widget.buttonItem!.type) {
+      case ContextMenuButtonType.cut:
+      case ContextMenuButtonType.copy:
+      case ContextMenuButtonType.paste:
+      case ContextMenuButtonType.selectAll:
+      case ContextMenuButtonType.delete:
+      case ContextMenuButtonType.custom:
+        return textWidget;
+      case ContextMenuButtonType.liveTextInput:
+        return SizedBox(
+          width: 13,
+          height: 13,
+          child: CustomPaint(
+            painter: _LiveTextIconPainter(color: _kToolbarTextColor.resolveFrom(context)),
+          ),
+        );
+    }
+  }
+}
+
+class _LiveTextIconPainter extends CustomPainter {
+  _LiveTextIconPainter({required this.color});
+
+  final Color color;
+
+  final Paint _painter = Paint()
+    ..strokeCap = StrokeCap.round
+    ..strokeJoin = StrokeJoin.round
+    ..strokeWidth = 1
+    ..style = PaintingStyle.stroke;
+
+  @override
+  void paint(Canvas canvas, Size size) {
+    _painter.color = color;
+    canvas.save();
+    canvas.translate(size.width / 2, size.height / 2);
+
+    final Offset origin = Offset(-size.width / 2.0, -size.height / 2.0);
+    // Path for the one corner.
+    final Path path = Path()
+      ..moveTo(origin.dx, origin.dy + 3.5)
+      ..lineTo(origin.dx, origin.dy + 1)
+      ..arcToPoint(Offset(origin.dx + 1, origin.dy), radius: const Radius.circular(1))
+      ..lineTo(origin.dx + 3.5, origin.dy);
+
+    // Rotate to draw corner four times.
+    final Matrix4 rotationMatrix = Matrix4.identity()..rotateZ(pi / 2);
+    for (int i = 0; i < 4; i += 1) {
+      canvas.drawPath(path, _painter);
+      canvas.transform(rotationMatrix.storage);
+    }
+
+    // Draw three lines.
```

Comment:
It is a common practice in the framework code to include the decimal point for doubles except maybe for "divide by two" like cases.